README.md

```diff
@@ -5,70 +5,58 @@
 [![GitHub Repo stars](https://img.shields.io/github/stars/enthought/comtypes?style=social)](https://github.com/enthought/comtypes/stargazers) [![GitHub forks](https://img.shields.io/github/forks/enthought/comtypes?style=social)](https://github.com/enthought/comtypes/network/members)  
 [![Tidelift Subscription](https://tidelift.com/badges/package/pypi/comtypes)](https://tidelift.com/subscription/pkg/pypi-comtypes?utm_source=pypi-comtypes&utm_medium=readme)
 
-## About
 
-`comtypes` is a lightweight `Python` [COM](https://learn.microsoft.com/en-us/windows/win32/com/component-object-model--com--portal) package, based on the [`ctypes`](https://docs.python.org/library/ctypes.html) FFI library.
+`comtypes` is a lightweight pure Python [COM](https://learn.microsoft.com/en-us/windows/win32/com/component-object-model--com--portal) package based on the [`ctypes`](https://docs.python.org/library/ctypes.html) foreign function library.
 
-`comtypes` allows to define, call, and implement custom and dispatch-based COM interfaces in pure `Python`.
+`comtypes` allows you to define, call, and implement custom and dispatch-based COM interfaces in pure Python.
 
-This package works on Windows only.
+`comtypes` requires Windows and Python 3.7 or later.
+- Version [1.2.1](https://pypi.org/project/comtypes/1.2.1/) is the last version to support Python 2.7 and 3.3–3.6.
+- `comtypes` does not work with Python 3.7.6 and 3.8.1 as reported in [GH-202](https://github.com/enthought/comtypes/issues/202), however, this bug has been fixed in Python 3.7 releases greater than or equal to 3.7.7 and Python 3.8 releases greater than or equal to 3.8.2.
+- Certain `comtypes` functions may not work correctly in Python 3.8 and 3.9 as reported in [GH-212](https://github.com/enthought/comtypes/issues/212). This bug has been fixed in Python >= 3.10.10 and >= 3.11.2.
 
-Available on `Python` 3.7-3.12.
-- [`comtypes==1.2.1`](https://pypi.org/project/comtypes/1.2.1/) is the last version supporting `Python` 2.7 and 3.3-3.6.
-- In `Python` 3.7.6 and 3.8.1, `comtypes` would not work as reported in [GH-202](https://github.com/enthought/comtypes/issues/202).  
-This bug has been fixed in `Python` >= 3.7.7 and >= 3.8.2.
-- In `Python` 3.8 and 3.9, some of `comtypes` functionalities may not work correctly as reported in [GH-212](https://github.com/enthought/comtypes/issues/212).  
-This bug has been fixed in `Python` >= 3.10.10 and >= 3.11.2.
-
-## Where to get it
-
-The source code is currently hosted on GitHub at:
-https://github.com/enthought/comtypes
-
-An installer for the latest released version is available at the [Python Package Index (PyPI)](https://pypi.org/project/comtypes).
+## Installation
 
+`comtypes` is available on [PyPI](https://pypi.org/project/comtypes) and may be installed with `pip`:
```

Comment:
```suggestion
`comtypes` is available on [PyPI](https://pypi.org/project/comtypes) and can be installed with `pip`:
```
If it is available on PyPI, it can be installed with `pip`. So I think 'can' is more appropriate than 'may'.
There are also other parts where 'will' or 'can' would be more suitable than 'may', so please consider them as well.

Reply:
Can and may are pretty interchangeable but I don't care much either way.

Reply:
I think that using 'may' indicates a possibility, but using 'can' or 'will' emphasizes that the action is actually possible or will definitely occur.

For example, I think that "By installing A, you can use B" is more appropriate than "By installing A, you may use B" when describing software functionality.

It may be my own bias, but I think that when 'may' is used in software descriptions, it typically implies "not limited to this" or "there is uncertainty".